recipes/cutlass/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -ex
+
+mkdir build
+cd build
+
+# avoid unnecessary components (i.e. examples; currently also not building profiler)
+# no GPU in CI, so disable building the tests...
+cmake \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_LIBRARY_OUTPUT_DIRECTORY="lib" \
+    -DCUTLASS_ENABLE_F16C=ON \
+    -DCUTLASS_ENABLE_CUDNN=ON \
+    -DCUTLASS_ENABLE_CUBLAS=ON \
+    -DCUTLASS_REVISION=${PKG_VERSION} \
+    -DCUTLASS_ENABLE_TOOLS=ON \
+    -DCUTLASS_ENABLE_LIBRARY=ON \
+    -DCUTLASS_ENABLE_PROFILER=OFF \
+    -DCUTLASS_ENABLE_TESTS=OFF \
+    -DCUTLASS_ENABLE_EXAMPLES=OFF \
+    ..  
```

Comment:
[You are required](https://conda-forge.org/docs/maintainer/knowledge_base.html#cuda-builds) to add conda-forge's CMAKE_ARGS variable.